integration_tests/e2e/order/monitoring-conditions/monitoringConditions.cy.ts

```diff
@@ -90,31 +94,61 @@ context('Monitoring conditions main section', () => {
-      cy.get('#orderTypeDescription-error').should('contain', 'You must select an option')
-      cy.get('#conditionType-error').should('contain', 'You must select an option')
-      cy.get('#monitoringRequired-error').should('contain', 'You must select an option')
-      cy.get('#startDate-error').should('contain', 'You must select an option')
-      cy.get('#endDate-error').should('contain', 'You must select an option')
+      cy.get('#orderType-error').should('contain', 'Order type is required')
+      cy.get('#conditionType-error').should('contain', 'Condition type is required')
+      cy.get('#monitoringRequired-error').should('contain', 'At least one monitoring type must be selected')
+      cy.get('#startDate-error').should('contain', 'Order start date is required')
+    })
+
+    it('after frontend validation passses, should show errors from API response', () => {
+      cy.task('stubCemoSubmitOrder', {
+        httpStatus: 400,
+        id: mockOrderId,
+        subPath: '/monitoring-conditions',
+        response: [
+          { field: 'orderType', error: 'Test error - order type' },
+          { field: 'orderTypeDescription', error: 'Test error - order type description' },
+          { field: 'conditionType', error: 'Test error - condition type' },
+          { field: 'updateMonitoringConditionsDto', error: 'Test error - monitoring required' },
+          { field: 'startDate', error: 'Test error - start date' },
+          { field: 'endDate', error: 'Test error - end date' },
+        ],
+      })
+      cy.signIn().visit(`/order/${mockOrderId}/monitoring-conditions`)
+      const page = Page.verifyOnPage(MonitoringConditionsPage)
+      cy.get('input[type="checkbox"]').check()
+      cy.get('select[name="orderType"]').select('immigration')
+      cy.get('select[name="orderTypeDescription"]').select('GPS Acquisitive Crime HDC')
+      cy.get('select[name="conditionType"]').select('License Condition of a Custodial Order')
+      cy.get('#startDate-day').type('27')
+      cy.get('#startDate-month').type('3')
+      cy.get('#startDate-year').type('2024')
+      cy.get('#startDate-hours').type('01')
+      cy.get('#startDate-minutes').type('02')
+      cy.get('#endDate-day').type('28')
+      cy.get('#endDate-month').type('4')
+      cy.get('#endDate-year').type('2025')
+      cy.get('#endDate-hours').type('03')
+      cy.get('#endDate-minutes').type('04')
+      page.form.saveAndContinueButton.click()
+      cy.get('#orderType-error').should('contain', 'Test error - order type')
+      cy.get('#orderTypeDescription-error').should('contain', 'Test error - order type description')
+      cy.get('#conditionType-error').should('contain', 'Test error - condition type')
+      cy.get('#monitoringRequired-error').should('contain', 'Test error - monitoring required')
+      cy.get('#startDate-error').should('contain', 'Test error - start date')
+      cy.get('#endDate-error').should('contain', 'Test error - end date')
```

Comment:
The MonitoringConditionsPage has helpers for you to do all of these kinds of things. So you could do 

```js
page.form.fillInWith({
  orderType: 'immigration'
  // add the other fields here
})
```
similarly
```js
page.form.startDateField.shouldHaveValidationMessage('Test error - order type')
```